mythtv/bindings/python/ttvdbv4/definitions.py

```diff
@@ -1,5 +1,11 @@
 # -*- coding: UTF-8 -*-
 
+# ---------------------------------------
```

Comment:
Should there be some sort of mythtv copyright info or yourname copyright. gpl/bsd/apache/mit?